sql/core/pom.xml

```diff
@@ -178,6 +178,17 @@
       <artifactId>htmlunit-driver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.lz4</groupId>
+      <artifactId>lz4-java</artifactId>
+      <scope>test</scope>
```

Comment:
Since we have this as a `test` scope, I'm wondering if we need to mention some additional info to the customers in the migration document. What do you recommend for the release, @viirya ?

Reply:
For sql/core, we don't include lz4-java originally. So I only have it as test scope for test purpose. For customers, they don't need to do any change. If they need lz4 codec, they just add lz4-java dependency as usual.

Reply:
So, this is the same behavior across all Spark versions (including 3.2.0 RC2), right?
> If they need lz4 codec, they just add lz4-java dependency as usual.

Reply:
Actually I found core module already has lz4-java (compile) dependency. So we don't need add it here again.

Reply:
It's great!